test/e2e/notifier/error/error.go

```diff
@@ -0,0 +1,75 @@
+package error
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/utils"
+	"github.com/infracloudio/botkube/test/e2e/env"
+	testutils "github.com/infracloudio/botkube/test/e2e/utils"
+)
+
+type context struct {
+	*env.TestEnv
+}
+
+func (c *context) testSKipErrorEvent(t *testing.T) {
+	// Modifying AllowedEventKindsMap to remove error event for pod resource
+	delete(utils.AllowedEventKindsMap, utils.EventKind{Resource: "v1/pods", Namespace: "all", EventType: "error"})
+
+	// Modifying AllowedEventKindsMap to add error event for only dummy namespace and ignore everything
+	utils.AllowedEventKindsMap[utils.EventKind{Resource: "v1/pods", Namespace: "dummy", EventType: "error"}] = true
+
+	tests := map[string]testutils.ErrorEvent{
+		"skip error event for resources not configured": {
+			// error event not allowed for Pod resources so event should be skipped
+			GVR:       schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"},
+			Kind:      "Pod",
+			Namespace: "test",
+			Specs:     &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "test-pod"}, Spec: v1.PodSpec{Containers: []v1.Container{{Name: "test-pod-container", Image: "tomcat:9.0.34"}}}},
+		},
+		"skip error event for namespace not configured": {
+			// error event not allowed for Pod in test namespace so event should be skipped
+			GVR:       schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"},
+			Kind:      "Pod",
+			Namespace: "test",
+			Specs:     &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "test-pod"}, Spec: v1.PodSpec{Containers: []v1.Container{{Name: "test-pod-container", Image: "tomcat:9.0.34"}}}},
+		},
+	}
+
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+			resource := utils.GVRToString(test.GVR)
+			// checking if error operation is skipped
+			isAllowed := utils.AllowedEventKindsMap[utils.EventKind{
+				Resource:  resource,
+				Namespace: "all",
+				EventType: config.ErrorEvent}] ||
+				utils.AllowedEventKindsMap[utils.EventKind{
+					Resource:  resource,
+					Namespace: test.Namespace,
+					EventType: config.ErrorEvent}]
```

Comment:
@Divya063 Let's use `utils.CheckOperationAllowed()` method we recently added to check if event allowed. Please rebase with the latest develop

Reply:
@PrasadG193 Sure!

Reply:
@PrasadG193 Done!

Reply:
@Divya063 did you miss committing the changes? I don't see this resolved